packages/cli/package/src/commands/provider/cc-collateral-withdraw.ts

```diff
@@ -1,43 +0,0 @@
-/**
- * Fluence CLI
- * Copyright (C) 2024 Fluence DAO
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, version 3.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-import { BaseCommand } from "../../baseCommand.js";
-import { collateralWithdraw } from "../../lib/chain/commitment.js";
-import { CHAIN_FLAGS, FLT_SYMBOL, CC_FLAGS } from "../../lib/const.js";
-import { initCli } from "../../lib/lifeCycle.js";
-
-// Deprecated alias for provider cc-finish
-export default class CCCollateralWithdraw extends BaseCommand<
```

Comment:
We don't need `cc-collateral-withdraw`?

Reply:
it still exists as an alias, it's an old naming, this file can be removed now after I updated oclif